### PR TITLE
Remaining style fixes

### PR DESCRIPTION
**Pull request by Pr0methean**
These are all the CheckStyle fixes that weren't in previous PRs.

### CODE REVIEW

**Comment by kashike** [comment]
ಠ_ಠ

File: src/main/java/net/glowstone/GlowWorld.java [hunk 1]
```diff
@@ -800,6 +821,13 @@ public boolean setSpawnLocation(int x, int y, int z) {
         return setSpawnLocation(new Location(this, x, y, z), true);
     }
 
+    /**
+     * Sets the spawn location of the world.
+     *
+     * @param newSpawn the new spawn location
+     * @param anchor if true, the spawn is never unloaded while the world is running
+     * @return
```
Comment (kaenganxt) [inline src/main/java/net/glowstone/GlowWorld.java:829]: true if the spawn location has changed

File: src/main/java/net/glowstone/generator/TheEndGenerator.java [hunk 2]
```diff
@@ -135,16 +135,16 @@ private void generateTerrainDensity(World world, int x, int z) {
 
         for (int i = 0; i < 3; i++) {
             for (int j = 0; j < 3; j++) {
-                double noiseHeight = 100.0D - Math.sqrt((x + i) * (x + i) + (z + j) * (z + j)) * 8.0D;
-                noiseHeight = Math.max(-100.0D, Math.min(80.0D, noiseHeight));
+                double nh = 100.0D - Math.sqrt((x + i) * (x + i) + (z + j) * (z + j)) * 8.0D;
```
Comment (kaenganxt) [inline src/main/java/net/glowstone/generator/TheEndGenerator.java:138]: The changes in this file are not correct.

File: src/main/java/net/glowstone/generator/decorators/nether/LavaDecorator.java [hunk 3]
```diff
@@ -38,12 +38,14 @@ public void decorate(World world, Random random, Chunk source) {
             return;
         }
         int netherrackBlockCount = 0;
-        int airBlockCount = 0;
```
Comment (kaenganxt) [inline src/main/java/net/glowstone/generator/decorators/nether/LavaDecorator.java:41]: Wrong changes in this file, too.

File: src/main/java/net/glowstone/generator/decorators/overworld/SurfaceCaveDecorator.java [hunk 4]
```diff
@@ -64,16 +64,15 @@ public void decorate(World world, Random random, Chunk c) {
 
     private void caveAroundRay(GlowBlock block, Random random) {
         int radius = random.nextInt(2) + 2;
-        final int blockX = block.getX();
```
Comment (kaenganxt) [inline src/main/java/net/glowstone/generator/decorators/overworld/SurfaceCaveDecorator.java:67]: Wrong changes in this file, too.

File: src/main/java/net/glowstone/generator/ground/MesaGroundGenerator.java [hunk 5]
```diff
@@ -71,12 +71,12 @@ public void generateTerrainColumn(ChunkData chunkData, World world, Random rando
         boolean colored = Math.cos(surfaceNoise / 3.0D * Math.PI) <= 0;
         double bryceCanyonHeight = 0;
         if (type == MesaType.BRYCE) {
-            int noiseX = (x & 0xFFFFFFF0) + (z & 0xF);
```
Comment (kaenganxt) [inline src/main/java/net/glowstone/generator/ground/MesaGroundGenerator.java:74]: Wrong changes in this file, too.

File: src/main/java/net/glowstone/generator/objects/OreVein.java [hunk 6]
```diff
@@ -43,22 +43,22 @@ public boolean generate(World world, Random random, int sourceX, int sourceY, in
             double radiusH = (Math.sin(i * (float) Math.PI / amount) + 1 * q + 1) / 2.0D;
             double radiusV = (Math.sin(i * (float) Math.PI / amount) + 1 * q + 1) / 2.0D;
             for (int x = (int) (originX - radiusH); x <= (int) (originX - radiusH); x++) {
-
-                // scale the center of x to the range [-1, 1] within the circle
-                double squaredNormalizedX = normalizedSquaredCoordinate(originX, radiusH, x);
```
Comment (kaenganxt) [inline src/main/java/net/glowstone/generator/objects/OreVein.java:48]: Wrong changes in this file, too.

File: src/main/java/net/glowstone/generator/objects/trees/AcaciaTree.java [hunk 7]
```diff
@@ -64,7 +64,7 @@ public boolean generate() {
                 twistCount--;
             }
 
-            Material material = blockTypeAt(centerX, loc.getBlockY() + y, centerZ);
```
Comment (kaenganxt) [inline src/main/java/net/glowstone/generator/objects/trees/AcaciaTree.java:67]: These changes are also not correct, right? (Also in the following files, all the blockAt method renames)

File: src/main/java/net/glowstone/generator/objects/trees/GenericTree.java [hunk 8]
```diff
@@ -164,31 +171,44 @@ public boolean generate() {
...
-     * @param y the y coordinate
-     * @param z the z coordinate
-     * @param newMaterial the new block type
-     * @param data the new block data
+     * @param x the absolute X coordinate
+     * @param y the absolute Y coordinate
+     * @param z the absolute Z coordinate
+     * @return the material of the given block
      */
-    protected void replaceIfAirOrLeaves(int x, int y, int z, Material newMaterial, int data) {
```
Comment (kaenganxt) [inline src/main/java/net/glowstone/generator/objects/trees/GenericTree.java:175]: Also this method and all it's uses.

File: src/main/java/net/glowstone/inventory/GlowPlayerInventory.java [hunk 9]
```diff
@@ -410,10 +410,7 @@ public int clear(Material type, MaterialData data) {
             if (stack != null && (type == null || stack.getType() == type) && (data == null || stack
                 .getData().equals(data))) {
                 setItem(i, InventoryUtil.createEmptyStack());
-                if (!InventoryUtil.isEmpty(stack)) {
```
Comment (kaenganxt) [inline src/main/java/net/glowstone/inventory/GlowPlayerInventory.java:413]: Also reverted from an earlier pull request.

**Review by kaenganxt — CHANGES_REQUESTED** [review]
It seems that there are several cases where this pull request reverts previous commits. I commented on some of the mixed up changes, but there may be more.

File: src/main/java/net/glowstone/generator/TheEndGenerator.java (hunk 2 above)
Comment (Pr0methean) [reply src/main/java/net/glowstone/generator/TheEndGenerator.java:138]: Reverted the file.

File: src/main/java/net/glowstone/generator/decorators/nether/LavaDecorator.java (hunk 3 above)
Comment (Pr0methean) [reply src/main/java/net/glowstone/generator/decorators/nether/LavaDecorator.java:41]: Reverted the package.

File: src/main/java/net/glowstone/generator/decorators/overworld/SurfaceCaveDecorator.java (hunk 4 above)
Comment (Pr0methean) [reply src/main/java/net/glowstone/generator/decorators/overworld/SurfaceCaveDecorator.java:67]: Reverted the package.

**Comment by Pr0methean** [comment]
Closing; will try using `git cherry-pick` into a fresh branch.

**Pull request closed without merging**